test/e2e/helpers_test.go

```diff
@@ -500,3 +464,43 @@ func getEnvValueInContainer(container *corev1.Container, name string) string {
 	}
 	return value
 }
+
+// getTemporaryKubeconfig dumps an environment's kubeconfig to a temporary file.
+func getTemporaryKubeconfig(t *testing.T, env environments.Environment) string {
+	t.Log("creating a tempfile for kubeconfig")
```

Comment:
We might as well dump the name of the file below in case it's needed for debugging ( I can see that it's removed in `t.Cleanup()` but who knows.

Reply:
[3164bf0](https://github.com/Kong/kubernetes-ingress-controller/pull/3378/commits/3164bf001de986c2d93112918c98fe4c73b9df0a)